src/main/java/seedu/address/ui/util/KeyListenerUtil.java

```diff
@@ -5,43 +5,58 @@
-    public static final KeyCombination UNDO = KeyCombination.valueOf(UndoCommand.COMMAND_HOTKEY);
-    public static final KeyCombination REDO = KeyCombination.valueOf(RedoCommand.COMMAND_HOTKEY);
-    public static final KeyCombination LIST = KeyCombination.valueOf(ListCommand.COMMAND_HOTKEY);
-    public static final KeyCombination OPEN_FILE = KeyCombination.valueOf(OpenCommand.COMMAND_HOTKEY);
-    public static final KeyCombination NEW_FILE = KeyCombination.valueOf(NewCommand.COMMAND_HOTKEY);
+    public static final KeyCombination FOCUS_PERSON_LIST_KEY_COMBINATION = KeyCombination.valueOf("Esc");
+    public static final KeyCombination FOCUS_PERSON_LIST_ALT_KEY_COMBINATION = KeyCombination.valueOf("Ctrl+Left");
+    public static final KeyCombination FOCUS_COMMAND_BOX_KEY_COMBINATION = KeyCombination.valueOf("Enter");
+    public static final KeyCombination FOCUS_RESULT_DISPLAY_KEY_COMBINATION = KeyCombination.valueOf("Ctrl+Right");
+    public static final KeyCombination DELETE_SELECTION_KEY_COMBINATION = KeyCombination.valueOf("Ctrl+D");
+    public static final KeyCombination CLEAR_KEY_COMBINATION = KeyCombination.valueOf(ClearCommand.COMMAND_HOTKEY);
+    public static final KeyCombination HISTORY_KEY_COMBINATION = KeyCombination.valueOf(HistoryCommand.COMMAND_HOTKEY);
+    public static final KeyCombination UNDO_KEY_COMBINATION = KeyCombination.valueOf(UndoCommand.COMMAND_HOTKEY);
+    public static final KeyCombination REDO_KEY_COMBINATION = KeyCombination.valueOf(RedoCommand.COMMAND_HOTKEY);
+    public static final KeyCombination LIST_KEY_COMBINATION = KeyCombination.valueOf(ListCommand.COMMAND_HOTKEY);
+    public static final KeyCombination OPEN_FILE_KEY_COMBINATION = KeyCombination.valueOf(OpenCommand.COMMAND_HOTKEY);
+    public static final KeyCombination NEW_FILE_KEY_COMBINATION = KeyCombination.valueOf(NewCommand.COMMAND_HOTKEY);
+    public static final KeyCombination ADD_KEY_COMBINATION = KeyCombination.valueOf(AddCommand.COMMAND_HOTKEY);
+    public static final KeyCombination EDIT_KEY_COMBINATION = KeyCombination.valueOf(EditCommand.COMMAND_HOTKEY);
+    public static final KeyCombination FIND_KEY_COMBINATION = KeyCombination.valueOf(FindCommand.COMMAND_HOTKEY);
+    public static final KeyCombination SELECT_KEY_COMBINATION = KeyCombination.valueOf(SelectCommand.COMMAND_HOTKEY);
+    public static final KeyCombination DELETE_KEY_COMBINATION = KeyCombination.valueOf(DeleteCommand.COMMAND_HOTKEY);
 
     public static final Set<KeyCombination> POSSIBLE_KEY_COMBINATIONS =
             new HashSet<>(Arrays.asList(
-                    FOCUS_PERSON_LIST,
-                    FOCUS_COMMAND_BOX,
-                    DELETE_SELECTION,
-                    CLEAR,
-                    HISTORY,
-                    UNDO,
-                    REDO,
-                    LIST,
-                    OPEN_FILE,
-                    NEW_FILE
+                    FOCUS_PERSON_LIST_KEY_COMBINATION,
```

Comment:
Associated constants (final variables) should be prefixed by a common type name. This is in the reverse order! See https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html